src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -329,6 +329,12 @@ double SolverInterfaceImpl::initialize()
 
   solverInitEvent.start(precice::syncMode);
 
+  for (MeshContext *meshContext : _accessor->usedMeshContexts()) {
+    if (meshContext->provideMesh) {
+      meshContext->mesh->removeDuplicates();
+    }
+  }
```

Comment:
Shouldn't this happen before mesh communication?

Reply:
Actually, before the complete repartitioning :see_no_evil:

Reply:
Fixed